hrjobcontract/scss/civihr/_bootstrap-variables.sass

```diff
@@ -112,7 +112,7 @@ $padding-xs-horizontal:     8px
 // $line-height-large:         1.33
 // $line-height-small:         1.5
 
-$border-radius-base:        0
+$border-radius-base:        2px
```

Comment:
this PR should be only about colors, why are we changing the border radius?

Reply:
We have done this changes to make input boxes and panels as per shoreditch which were slightly different.